### PR TITLE
261 test and member related issues

### PR DESCRIPTION
**Pull request by yash6102000**
- When deleting test, popup is not coming.(fixed)
- We can create a test just giving space in name and description field.(fixed)
- If a test with 0 invites is deleted, no need to show it in the results table(fixed)
- Role is not reseting on opening add member model again.(fixed)
- on reset password pop up input fields are not resetting after opening it again(fixed)

### CODE REVIEW

File: app/components/results/GroupByTests.tsx [hunk 1]
```diff
@@ -26,6 +26,14 @@ const GroupByTests = () => {
   ]
   const candidateTestData = useLoaderData()
   const candidateTests = candidateTestData.candidateTest
+  const newArray = candidateTests.filter(function (
```
Comment (richa-copods) [inline app/components/results/GroupByTests.tsx:29]: Please use arrow function to maintain the code syntax consistency

Comment (richa-copods) [inline app/components/results/GroupByTests.tsx:29]: Please give proper name instead of "newArray"

File: app/routes/tests/index.tsx [hunk 2]
```diff
@@ -105,9 +105,22 @@ export default function Tests() {
   const { t } = useTranslation()
 
   const data = useLoaderData() as unknown as LoaderData
+  const testActionData = useActionData() as ActionData
   if (t(data.status as string) != t('statusCheck.success')) {
     toast.warn(t('statusCheck.commonError'))
   }
+  console.log(testActionData)
```
Comment (richa-copods) [inline app/routes/tests/index.tsx:112]: Please remove this

**Review by richa-copods — CHANGES_REQUESTED** [review]

**Comment by anuragP04** [comment]
The description is missing in this PR, Please add whatever changes you have made. @yash6102000

File: app/routes/tests/index.tsx [hunk 3]
```diff
@@ -5,27 +5,27 @@ import type { LoaderFunction, ActionFunction } from '@remix-run/node'
...
 
 type LoaderData = {
   tests: Awaited<ReturnType<typeof getAllTests>>
   status?: string | undefined
 }
 export type ActionData = {
   errors?: {
-    status?: string
-    check?: Date
+    status: number
```
Comment (richa-copods) [inline app/routes/tests/index.tsx:23]: Can we please name it as "statusCode"

**Review by richa-copods — CHANGES_REQUESTED** [review]

**Review by richa-copods — APPROVED** [review]

**Pull request merged**